src/plugins/appleMusic.desktop/index.tsx

```diff
@@ -243,17 +245,17 @@ export default definePlugin({
 
             name: customFormat(settings.store.nameString, trackData),
             details: customFormat(settings.store.detailsString, trackData),
-            state: customFormat(settings.store.stateString, trackData),
+            state: isARadioStation ? undefined : customFormat(settings.store.stateString, trackData),
 
-            timestamps: (settings.store.enableTimestamps ? {
+            timestamps: (trackData.playerPosition && trackData.duration) ? (settings.store.enableTimestamps ? {
```

Comment:
Why the nested ternary? Putting all the conditions together is functionally equivalent. Also, `playerPosition` being 0 would be a perfectly valid value to put in timestamps.

```suggestion
            timestamps: (Number.isNaN(trackData.duration) && settings.store.enableTimestamps ? {
```